src/azul/health.py

```diff
@@ -151,3 +159,46 @@ def _lambda(self, lambda_name) -> JSON:
             return {
                 'up': up,
             }
+
+
+class HealthController(Health):
+
+    def __init__(self, lambda_name: str, keys: str = None, request_path: str = '/health'):
+        super().__init__(lambda_name)
+        self.keys = keys
+        self.request_path = request_path
+        self.storage_service = StorageService()
+
+    def response(self):
+        if self.keys == 'basic':
+            return {
+                'up': True,
+            }
+        elif self.keys == 'cached':
+            try:
+                health_object = json.loads(self.storage_service.get(f'health/{self.lambda_name}'))
+            except self.storage_service.client.exceptions.NoSuchKey:
+                return Response(status_code=500,
+                                body=json.dumps({'Message': 'Object key does not exist in S3'}))
```

Comment:
```suggestion
                                body=json.dumps({'Message': 'Cached health object does not exist'}))
```

Note the linguistic correspondence between the two error messages.